.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,32 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+2. Ensure you have added or run the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+3. Understand the relationship beteween JIRAs to file issues and PRs to propose some changes for them, as documented in guidelines.
+4. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][SPARK-XXXX] Your PR title ...'
+5. Update your PR description whenever it is needed.
+6. Provide a minimised example to reproduce the issue if possible.
+-->
 
-(Please fill in changes proposed in this fix)
+### What changes were proposed in this pull request?
```

Comment:
just a curiosity: why are we changing the title level?

Reply:
Just it looked too big to me. I don't mind changing back.

Reply:
no, I was just curious, I am fine whatever :)

Reply:
Once it is merged it doesn't provide any difference so not a big deal.